### ISSUE TITLE
OptiFine

### ISSUE THREAD

**Issue by PPPaluch** [blackd/Inventory-Profiles#340]
### Have you asked for help on discord?

NO

### Minecraft Version/s

1.19.2

### Modloader

Fabric

### Modloader Version

fabric-loader-0.14.21-1.19.2
fabric-api-0.76.01.19.2

### Operating System

Windows

### List of all other installed mods

fabric-api-0.76.01.19.2
fabric-language-kotlin-1.9.5+kotlin.1.8.22
InventoryProfilesNext-fabric-1.19-1.10.4
libIPN-fabric-1.19-3.0.2
optifabric-1.13.24
OptiFine_1.19.2_HD_U_I1

### Describe the problem

I'm starting minecraft with OptiFine - instant game crash
I'm starting minecraft without OptiFine - all working fine
I'm starting minecraft with OptiFine, but without Inventory Profiles Next - all working fine

In short it doesn't work with OptiFine... WHY?...

### Steps to reproduce

Just add OptiFine to the mods folder

### I have tried to reproduce the problem with only or without "Inventory Profiles Next" and its dependencies installed.

- [X] I have tested without 'Inventory Profiles Next' and its dependencies
- [X] I have tested with only 'Inventory Profiles Next' and its dependencies and no other mods

**Comment by blackd** [comment]
this is a know issue and was described in the release notes of 1.10.3
so far there is no answer in the optifabric tracker

**Comment by blackd** [comment]
I'm looking into working around it but no luck so far

**Comment by blackd** [comment]
https://github.com/Chocohead/OptiFabric/issues/1063
